tests/testnative.nim

```diff
@@ -24,8 +24,7 @@ import
   testtortransport,
   testwstransport,
   testquic,
-  testmemorytransport,
-  transports/tls/testcertificate
```

Comment:
`transports/tls/testcertificate` should be kept here.

add `{.used.}` to top of testcertificate.nim